### ISSUE TITLE
chrome doesn't resize

### ISSUE THREAD

**Issue by sharmarahul** [alisterscott/webdriver-user-agent#24]
> irb
    > require 'webdriver-user-agent'
    > driver = Webdriver::UserAgent.driver(browser: :chrome) 

doesn't open an iphone size window.

**Comment by samnissen** [comment]
tl;dr: I can replicate. Chrome lies about `innerWidth` and `resizeTo` is not supported. Crazy workaround of using experimental devtools access could work, needs exploratory testing. Comes with onerous dependency of starting initializing Chrome with a specific flag.

```bash
$ irb
```
```ruby
> require 'webdriver-user-agent'
 => true
> driver = Webdriver::UserAgent.driver(browser: :chrome)
 => #<Selenium::WebDriver::Driver:0x6907769461e2bc2e browser=:chrome>
> driver.execute_script('return window.innerWidth')
 => 375
> driver.execute_script('return document.documentElement.clientWidth')
 => 1050
> driver.execute_script("return document.getElementsByTagName('body')[0].clientWidth")
 => 1050
> driver.execute_script("document.documentElement.clientWidth = 375")
 => nil
> driver.execute_script("document.getElementsByTagName('body')[0].clientWidth = 375")
 => nil
> driver.execute_script("window.resizeTo(375,375)")
 => nil
> driver.execute_script("return document.getElementsByTagName('body')[0].clientWidth")
 => 1050
```

I tried this with specific agents -- no change. After some Googling it turns out Chrome has not supported resizing for some time: https://code.google.com/p/chromium/issues/detail?id=2091

BUT there are work arounds: http://webmasters.stackexchange.com/questions/23129/how-do-you-set-the-width-of-the-browser-using-google-chrome

Tried one:

```ruby
> driver.execute_script("window.open('javascript:window.resizeTo(1024,768)'")
Selenium::WebDriver::Error::UnknownError: unknown error: Runtime.evaluate threw exception: SyntaxError: Unexpected token }
```

I also tried with escaped double quotes, no quotes, double escaped double quotes -- different error messages, but no effect. So no to that, I guess? However, there is another option that I cannot test at the moment:

1. Start browser with the experimental flag `--enable-experimental-extension-apis` https://developer.chrome.com/apps/experimental (This might be outside the scope of the gem and up to the user to execute correctly. Obviously it would need to be documented as a dependency. And it would suck.)
2. Access the console via `chrome.experimental.devtools` http://stackoverflow.com/a/6822670
3. Then run the following `var myTestWindow = window.open(window.location); myTestWindow.resizeTo(1024,768);`

I'll try to test this later.

My stack:
- Chrome 41 
- OS X 10.10
- webdriver-user-agent 7.2
- ruby 2.1.2
